ui/pages/confirmations/hooks/gas/useIsGaslessLoading.test.ts

```diff
@@ -0,0 +1,125 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useSelector } from 'react-redux';
+import { useConfirmContext } from '../../context/confirm';
+import { getUseTransactionSimulations } from '../../../../selectors';
+import { useIsInsufficientBalance } from '../useIsInsufficientBalance';
+import { useIsGaslessSupported } from './useIsGaslessSupported';
+import { useIsGaslessLoading } from './useIsGaslessLoading';
+
+jest.mock('../../context/confirm');
+jest.mock('./useIsGaslessSupported');
+jest.mock('../useIsInsufficientBalance');
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: jest.fn(),
+}));
+
+const mockedUseConfirmContext = jest.mocked(useConfirmContext);
+const mockedUseIsGaslessSupported = jest.mocked(useIsGaslessSupported);
+const mockedUseIsInsufficientBalance = jest.mocked(useIsInsufficientBalance);
+const mockedUseSelector = useSelector as jest.Mock;
+
+describe('useIsGaslessLoading', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const setup = ({
```

Comment:
Minor, for hook unit tests, I usually create a `runHook` function that includes this state setup, plus the `renderHook(() => useIsGaslessLoading())` also.

Reply:
Removed mocked selectors and changed the setup function to the pattern `runHook`.